docs/up-ga/reads/v1.md

```diff
@@ -11,7 +11,7 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-  Total views on pages where the work can be read, as determined by Google Analytics. To count multiple reads on the same page by the same user, there must be a gap of at least 30 minutes from the last read of that page, even if that previous read was not counted.
+  Total views on pages where the work can be read, as determined by Google Analytics. To count multiple reads on the same page by the same user, there must be a gap of at least 30 minutes from the last read of that page, even if that previous read was not counted. NOTE: A read that occurs 2 hours after the first read will not be counted. See the last entry in the table below; it should be counted because it is 30 minutes after the last read, but because it is 2 hours after the first read, it is not counted.
```

Comment:
Total views on pages where the work can be read, as determined by Google Analytics. To count multiple reads on the same page by the same user, there must be a period of at least 30 minutes of inactivity on the website, by the user.

Note: The user's activity is logged in a session. The example below considers one type of interaction on a single page; however, a single session can contain interactions involving page views, downloads, and reads (even across different pages on the website). These interactions are logged within the same session if they occur before the session timeout. The user's session must end before another read of a specific page can be counted for that user (i.e. as part of an different session).